src/api-types.ts

```diff
@@ -269,3 +269,79 @@ export interface TuneOutput {
 export interface TuneMethodsOutput {
   results: { id: string; name: string }[];
 }
+
+// PROMPT TEMPLATES
+
+export const PromptTemplateBaseInputSchema = z.object({
+  id: z.string(),
+});
+
+export type PromptTemplateBaseInput = z.input<
+  typeof PromptTemplateBaseInputSchema
+>;
+
+export const PromptTemplateCreateInputSchema = z.object({
+  name: z.string(),
+  value: z.string(),
+});
+
+export type PromptTemplateCreateInput = z.input<
+  typeof PromptTemplateCreateInputSchema
+>;
+
+export const PromptTemplateUpdateInputSchema = z.union([
+  PromptTemplateBaseInputSchema,
+  PromptTemplateCreateInputSchema.partial(),
+]);
+export type PromptTemplateUpdate = z.input<
+  typeof PromptTemplateUpdateInputSchema
+>;
+
+export const PromptTemplatesInputSchema = z.object({
+  limit: z.number().int().min(1),
+  offset: z.number().int().min(0),
+});
+export type PromptTemplatesInput = z.input<typeof PromptTemplatesInputSchema>;
```

Comment:
We're exporting api schemas for body only, so the naming is confusing. We can support "parameter schemas" in the future though. Remove this for the time being.